src/main/java/org/folio/models/EncumbrancesProcessingHolder.java

```diff
@@ -113,10 +104,7 @@ public List<Transaction> getEncumbrancesToUnreleaseAfter() {
 
   public int getAllEncumbrancesQuantity() {
     return encumbrancesForCreate.size() + encumbrancesForUpdate.size()
-        + encumbrancesForRelease.size() + encumbrancesForUnrelease.size();
-  }
-
-  public List<Transaction> getEncumbrancesFromStorage() {
-    return ImmutableList.copyOf(encumbrancesFromStorage);
+        + encumbrancesForRelease.size() + encumbrancesForUnrelease.size()
+        + encumbrancesForDelete.size();
```

Comment:
can you please explain more about this change, it was some bug for it previously?